### DISCUSSION TITLE
Need assistance on which data variable to be edited for SMAP_L3 - 9km for LIS DA

### DISCUSSION THREAD

**Discussion by MariRiba** [NASA-LIS/LISF#1419] [Q&A]
Hi, I want to edit grid points of SMAP L3 data of 9km resolution, and run LIS DA with the same.

I went through the codes for the same, but there are two variables of soil moisutre, namely soil moisture and soil_moisture_pm.

I could not find any more leads.

Please help

**Comment by dmocko** [comment]
Hi @MariRiba 

Can you please provide more details about exactly what product you are attempting to use?

Please provide the exact product name, provide a sample file name, and also attach your lis.config file.

Thanks,

David
